suites/nautilus/rgw/tier-2_rgw_s3tests.yaml

```diff
@@ -34,7 +34,7 @@ tests:
       desc: test deploying of cluster setup using ceph-ansible
       destroy-cluster: false
       abort-on-fail: true
-      polarion-id: CEPH-83571467
+      polarion-id: CEPH-83571467, CEPH-83572684
```

Comment:
It is recommended to have a single polarion id for direct mapping as it helps when TFA is involved.

Reply:
Hi Ameensa,

As we need to add the polarion ID to test scripts if already automated, what can be done here?
This testcase applies to 2 polarion cases...Any suggestions?

Reply:
We could check if CEPH-83571467 is covered else where then update a single polarion id here

Reply:
I have remove CEPH-83571467 as this case was covered/updated in many other test scripts